e2e/mocks.js

```diff
@@ -1,5 +1,3 @@
 const path = require('path')
 const electron = require('electron')
-electron.dialog.showOpenDialog = (opts, cb) => {
-  cb([path.join(__dirname, 'support', 'readable.pdf')])
-}
+electron.dialog.showOpenDialog = (opts, cb) => cb([path.join(__dirname, 'support', 'readable.pdf')])
```

Comment:
Why is this in one line? Two things happen in the logic, so it would be better to keep it on two lines.